data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2471 @@
+
+outfit "spin4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" 5
+		"lifetime" 40
+		"submunition" "track4 print left"
+
+outfit "track4 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "track4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "asteroid laser 1"
+	weapon
+		sprite "projectile/asteroid laser"
+			"frame rate" 60
+			"random start frame"
+		sound "zapper"
+		"hit effect" "skylance impact"
+		"target effect" "asteroid target"
+		"inaccuracy" .3
+		"velocity" 200
+		"lifetime" 1
+		"reload" 1
+		"discharge damage" 1.7
+		"hull damage" 5.9
+
+effect "asteroid hit"
+	sprite "effect/point defense hit"
+		"frame rate" 13
+		"no repeat"
+	lifetime 1
+	"velocity scale" 0
+
+effect "asteroid target"
+	sprite "effect/finisher sparks"
+		"random start frame"
+		"frame rate" 20
+	"lifetime" 15
+	"random angle" 180
+
+effect "asteroid circle trail"
+	sprite "effect/asteroid circle trail"
+		"no repeat"
+		"frame time" 4
+	"lifetime" 20
+	"random velocity" 5
+	"random spin" 10
+	"velocity scale" 0.3
+
+outfit "Asteroid Weapon Large 3"
+	"display name" "???"
+	category "Guns"
+	cost 98713449
+	thumbnail "outfit/asteroid weapon 5"
+	"mass" 16
+	"outfit space" -16
+	"weapon capacity" -16
+	"gun ports" -1
+	"asteroid mount 3" -1
+	weapon
+		sprite "projectile/asteroid twin projectile"
+			"frame rate" 10
+			"random start frame"
+		"velocity" 20
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+			"facing" 30
+		"submunition" "asteroid missile"
+			"facing" -30
+		"submunition" "magic deployer1"
+		"homing" 1
+		"optical tracking" 1
+		"turn" 0
+		"split range" 180
+		"reload" 100
+		"firing heat" 5
+	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy created by this weapon's fire continuously releases projectiles which home in onto enemies, until finally exploding and breaking up into an enormous barrage of similar projectiles. These attributes make it very good at overwhelming anti-missiles enough to render them useless, then impacting a fair share of damage while they are down."
+
+outfit "asteroid missile"
```

Comment:
You have two submunitions defined with this name - this one, and the one at the end of the definition of `Asteroid Weapon Large`.

Reply:
I mentioned this earlier, but I changed the first one to "asteroid launcher". The two submunitions are similar, but it's easier to balance each separately